packages/cozy-jobs-cli/package.json

```diff
@@ -9,12 +9,10 @@
   },
   "license": "MIT",
   "bin": {
-    "cozy-konnector-dev": "bin/cozy-konnector-dev",
-    "cozy-konnector-standalone": "bin/cozy-konnector-standalone",
-    "cozy-konnector-shell": "bin/cozy-konnector-shell",
-    "cozy-run-shell": "bin/cozy-run-shell",
-    "cozy-run-standalone": "bin/cozy-run-standalone",
-    "cozy-run-dev": "bin/cozy-run-dev"
+    "cozy-konnector-shell": "src/konnector-shell",
```

Comment:
- src/konnector-shell.js
- src/konnector-dev.js
- src/konnector-standalone.js

+ `chmod +x` for these files